assets/airports/efhk.json

```diff
@@ -0,0 +1,1361 @@
+        "EMPEV": ["N60d14m17.68", "E24d44m25.00"],
+        "LAVDI": ["N60d14m14.15", "E24d46m26.73"],
+        "TIXED": ["N60d25m44.59", "E25d11m49.01"],
+        "HAGIS": ["N60d12m31", "E24d54m00"],
+        "HAKKI": ["N60d16m43", "E25d05m30"],
+        "KOLIS": ["N60d27m04", "E25d01m07"],
+        "LILJA": ["N60d19m26", "E25d12m31"],
+        "LINTU": ["N60d22m47", "E24d41m04"],
+        "OGELI": ["N60d13m51", "E24d59m16"],
+
+        "NIPIB": ["N59d40m57", "E23d00m48"],
+        "ODESO": ["N60d19m34", "E22d39m02"],
+        "PEKUX": ["N60d24m19", "E22d37m32"],
+        "IPMOT": ["N60d27m11", "E23d10m35"],
+        "AMROT": ["N60d29m38", "E23d42m33"],
+        "ERNUT": ["N61d17m10", "E25d16m47"],
+        "OTPAL": ["N61d16m08", "E25d40m32"],
+        "ROKVI": ["N60d51m39", "E27d01m23"],
+        "ARLOM": ["N60d47m25", "E27d43m53"],
+        "LEDUN": ["N59d58m30", "E26d06m42"],
+        "ARSAM": ["N59d26m51", "E25d21m25"],
+        "BEKNA": ["N59d31m40", "E24d56m05"],
+        "GEPTA": ["N59d34m48", "E25d29m14"],
+        "SORPA": ["N59d37m01", "E24d50m49"],
+
+        "PEPIG": ["N60d08m29", "E22d35m59"],
+        "RATMU": ["N60d14m16", "E26d32m09"],
+        "INLOG": ["N60d10m18", "E26d56m47"],
+        "OLEMA": ["N60d14m00", "E27d28m30"],
+        "RATLA": ["N60d51m30", "E28d20m18"],
+        "MUSTI": ["N59d45m07", "E24d22m47"],
+        "TLL": ["N59d24m41", "E24d50m18"],
+        "TUSBI": ["N61d04m05", "E26d18m23"],
+        "UMUGI": ["N60d02m14", "E22d41m41"],
+        "NISVI": ["N61d07m34", "E24d02m55"],
+        "ABSER": ["N59d53m50", "E22d59m48"],
+        "PISOX": ["N59d46m41", "E22d45m04"],
+        "DIPUM": ["N61d13m00", "E24d55m08"],
+        "TN471": ["N59d34m15.4", "E25d02m47.4"],
+        "TN903": ["N59d37m11.0", "E24d26m19.1"],
+        "TN425": ["N59d48m10.3", "E24d53m11.0"],
+
+        "_22RDME25": ["N60.3089654", "E24.8951079"]
+    },
+    "runways":[
+        {
+            "name": ["04L", "22R"],
+            "name_offset": [[0.9, 1], [0, 0]],
+            "end": [["N60d18m46.61", "E24d54m13.93", "133.6ft"], ["N60d19m53.42", "E24d56m40.90", "179.9ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["04R", "22L"],
+            "name_offset": [[0.9, 1], [-0.8, -1]],
+            "end": [["N60d18m40.65", "E24d56m10.94", "153.2ft"], ["N60d19m50.49", "E24d58m44.73", "148.6ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["15", "33"],
+            "name_offset": [[0, 0], [-0.7, 1]],
+            "end": [["N60d19m48.99", "E24d57m52.19", "162.9ft"], ["N60d18m25.44", "E24d59m17.83", "147.1ft"]],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "ASTIV": {
+            "icao": "ASTIV",
+            "name": "ASTIV",
+            "suffix": {"22R":"3N"},
+            "rwy": {
+                 "22R": ["_22RDME25", "HK451", ["ASTIV", "S250-"]]
+            },
+            "exitPoints": {
+                "DOBAN": ["DOBAN"],
```

Comment:
See "EXPECTED DEPARTURE PROCEDURE" in:

https://www.ais.fi/ais/aip/pdf/aerodromes/EF_AD_2_EFHK_EN.pdf

However, I (as a layman) don't really know if they should be done like that or not (as transitions or whether the exitpoints should just be a part of route in spawnpatterns after "..").

In addition, (more relevant to the spawn patterns) there is information on how routes proceed onwards from those TMA exit points in AIP (ENR 3.5). I also found from somewhere that RENKU is available only for flights to Tallinn but I cannot now immediately come across that again (perhaps it was from some Estonian documentation).